src/types/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
From https://vitejs.dev/guide/features#client-types. It seems we either need this file or `types: ["vite/client"]` in `tsconfig.json`, so having both seems unnecessary. I've tested that after removing this file I still get types for `import.meta.env` as expected.

Reply:
Oh nice, I didn't knew that the `vite/client` type is already there. Thanks!